src/System.Device.Gpio/Interop/Unix/libgpiod/Iterop. gpiod_line_event_wait.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
there is something wrong with the name of this file

Reply:
yeah it starts with `Iterop.` not `Interop. `

Reply:
github also shows the space in between dot and g (in gpiod)